src/tqec/enums.py

```diff
@@ -43,3 +43,12 @@ class PlaquetteOrientation(Enum):
     LEFT = auto()
     DOWN = auto()
     UP = auto()
+
+
+class PlaquetteOrigin(Enum):
+    """Defines the origin of a plaquette."""
+
+    CENTRAL = auto()
+    FIRST = auto()
+    TOP_LEFT = auto()
+    USER_DEFINED = auto()
```

Comment:
Wouldn't it be simpler to let the user define its origin implicitly? Basically, do we really need to have that enumeration? 
The current behaviour is to accept qubit coordinates as the user provides them. If the user provides qubits like in [`SquarePlaquette`](https://github.com/QCHackers/tqec/blob/4f047700c59514d17a6248c821119179da9126b5/src/tqec/plaquette/plaquette.py#L48):
https://github.com/QCHackers/tqec/blob/4f047700c59514d17a6248c821119179da9126b5/src/tqec/plaquette/plaquette.py#L48-L93
then the origin is implicitly `TOP_LEFT`. If the user provides qubits like
```python
class CenteredSquarePlaquette(Plaquette):
    def __init__(self, circuit: ScheduledCircuit) -> None:
        """Represents a centered square QEC plaquette

        By convention, the qubits are sorted as depicted in the text below:
        ```text
        |---------|
        | 1     2 |
        |         |
        | 3     4 |
        |---------|
        ```
        Sub-classes of this class should take that into account to apply operations
        on the correct qubits.
        This ordering is not to be confused with the temporal ordering of multi-qubit
        gates, for example in surface codes. It is a **qubit** ordering and has **no
        relation** with a temporal ordering.

        :param circuit: scheduled quantum circuit implementing the computation that the
            plaquette should represent.
        """
        super().__init__(self.get_data_qubits() + self.get_syndrome_qubits(), circuit)

    _data_qubits: list[PlaquetteQubit] = [
        PlaquetteQubit(Position(-1, -1)),
        PlaquetteQubit(Position(1, -1)),
        PlaquetteQubit(Position(-1, 1)),
        PlaquetteQubit(Position(1, 1)),
    ]
    _syndrome_qubits = [PlaquetteQubit(Position(0, 0))]
    # Everything else the same
```
then the origin is implicitly `CENTER`. 

Do we care about / need the information provided by this enumeration?

Reply:
Also, this method would simplify (in my opinion) user input.

Reply:
My idea was to have this somewhat flexible, but simple setting would be fine with me. I mainly did this to see what would be working the best in the beginning.

Reply:
Do you have a situation in mind where the simple setting above is not enough and we actually need the information provided by `PlaquetteOrigin`? I tried to find one, but could not.